wallet/src/de/schildbach/wallet/ui/dashpay/work/SendInviteWorker.kt

```diff
@@ -67,12 +84,17 @@ class SendInviteWorker(context: Context, parameters: WorkerParameters)
         }
 
         return try {
-            val blockchainIdentity = PlatformRepo.getInstance().getBlockchainIdentity()!!
+            val blockchainIdentity = platformRepo.getBlockchainIdentity()!!
             val cftx = platformRepo.createInviteFundingTransactionAsync(blockchainIdentity, encryptionKey)
+            val dashPayProfile = platformRepo.getLocalUserProfile()
+            val dynamicLink = createDynamicLink(dashPayProfile!!, cftx.creditBurnIdentityIdentifier.toStringBase58())
```

Comment:
The link should have the ctfx.txid.toString() and the cftx.creditBurnPublicKey.privateKeyHex and possibly other information like IS lock info.  We can probably change this with the accept invite related stories.